5.37.0/upgradeLegacyRteToCmsFeatureFlag.ts

```diff
@@ -0,0 +1,56 @@
+import { Files } from "../utils";
+import { Project } from "ts-morph";
+import { Context } from "../types";
+import { featureFlagExist } from "../utils/featureFlagExist";
+import { addFeatureFlag } from "../utils/addFeatureFlag";
+
+interface Params {
+    files: Files;
+    project: Project;
+    context: Context;
+}
+
+export const upgradeLegacyRteToCmsFeatureFlag = async (params: Params) => {
+    const { project, files, context } = params;
+
+    context.log.info(
+        "Start migrating feature flag for legacy Rich Text editor support in the Headless CMS app."
+    );
+
+    const webinyProjectFile = files.byName("./webiny.project.ts");
+    const source = project.getSourceFile(webinyProjectFile.path);
+
+    const FEATURE_FLAG_NAME = "allowCmsLegacyRteInput";
+
+    if (!source) {
+        context.log.error(
+            "Feature flag migration is canceled, file 'webiny.project.ts' can be found."
```

Comment:
Instead of `'webiny.project.ts'` just do `%s` and add the `webiny.project.ts` (without single quotes) as a new argument to the `context.log.error` call. That will make the word nicely highlighted.